x/bounty/keeper/grpc_query.go

```diff
@@ -1 +1,88 @@
 package keeper
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/query"
+
+	"github.com/shentufoundation/shentu/v2/x/bounty/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+// Hosts implements the Query/Hosts gRPC method
+func (k Keeper) Hosts(c context.Context, req *types.QueryHostsRequest) (*types.QueryHostsResponse, error) {
```

Comment:
need implementation

Reply:
yes, but currently there is no host data to query